libs/components/pages/src/lib/modules/needs-attention/needs-attention.component.scss

```diff
@@ -9,30 +9,27 @@
   display: flex;
   flex-wrap: nowrap;
   align-items: center;
+  cursor: pointer;
+  text-align: left;
+  width: 100%;
```

Comment:
I'm not sure you need this, either, if `display` is `flex`.

Reply:
The `text-align` and `width` styles are necessary for the button.

Reply:
If it were `display: block flex` it would have default width `100%` but `button` has a default `<display-outside>` of `inline` and [MDN says it's too soon](https://developer.mozilla.org/en-US/docs/Web/CSS/display#which_syntax_should_you_use_now) to adopt the newer syntax.